src/sasquatchbackpack/cli.py

```diff
@@ -180,10 +183,30 @@ def usgs_earthquake_data(
             click.echo(result)
         click.echo("------")
     else:
-        click.secho("SUCCESS", fg="orange")
+        click.secho("SUCCESS! (kinda)", fg="yellow")
         click.echo("------")
         click.echo("No results found for the provided criteria :(")
         click.echo("------")
+        return
+
+    click.confirm("Do you want to post the above data?", abort=True)
```

Comment:
My main comment on the CLI is to avoid interactive prompts as we'll need to run from a docker container later. 
I suggest implementing a `--dry-run` option instead:

```python
@click.option(
    "--dry-run",
    is_flag=True,
    help="Perform a trial run with no data being sent to Kafka.",
)
```
and then:

```python
        if dry_run:
            click.echo("Dry run mode: No data will be sent to Kafka.")
            return
```